arch/risc-v/src/k210/k210_cpustart.c

```diff
@@ -112,6 +112,25 @@ void k210_cpu_boot(int cpu)
   showprogress('b');
   DPRINTF("CPU%d Started\n", this_cpu());
 
+#ifdef CONFIG_STACK_COLORATION
+    {
+      FAR struct tcb_s *tcb = this_task();
+      register uint32_t *ptr;
+      register int i;
+
+      /* If stack debug is enabled, then fill the stack with a recognizable
+       * value that we can use later to test for high water marks.
+       */
+
+      for (i = 0, ptr = (uint32_t *)tcb->stack_alloc_ptr;
+           i < tcb->adj_stack_size;
+           i += sizeof(uint32_t))
+        {
+          *ptr++ = STACK_COLOR;
+        }
+    }
```

Comment:
can replace with riscv_stack_color?

Reply:
> can replace with riscv_stack_color?

@xiaoxiang781216 
OK. I will try it this evening.

Reply:
@xiaoxiang781216 
Done.